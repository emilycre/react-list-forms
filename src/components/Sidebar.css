.Sidebar {
  display: flex;
  justify-content: space-around;
  flex-direction: column;
  width: 25vw;
  height: 100vh;
  padding: 1rem;

  background-color: black;
  color: gray;

  text-align: center;
  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;

  a {
    display: block;

    color: gray;

    font-size: 1.5rem;
    text-decoration: none;
  }
}
