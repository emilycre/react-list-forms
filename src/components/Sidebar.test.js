import React from 'react';
import { shallow } from 'enzyme';
import Sidebar from './Sidebar';

describe('Sidebar Component', () => {
  it('Renders a Sidebar', () => {
    const wrapper = shallow(
      <Sidebar>
        <a href="Hello.">Testing...</a>
      </Sidebar>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
