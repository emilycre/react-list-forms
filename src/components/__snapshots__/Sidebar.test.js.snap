// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar Component Renders a Sidebar 1`] = `
<nav
  className="Sidebar"
>
  <a
    href="Hello."
  >
    Testing...
  </a>
</nav>
`;
